content/learn/05.communication/08.bluetooth/bluetooth.md

```diff
@@ -0,0 +1,356 @@
+
+**Common Standard Profiles**:
+
+Bluetooth® LE includes a range of standard profiles that simplify the development of BLE applications. Some of the most well-known standard profiles include:
+
+- Battery Service: The Battery Service provides information about the battery level of a device. It typically includes a Battery Level characteristic that central devices can read to monitor the battery status of a peripheral device, such as a wireless headset or smartwatch.
+
+- Heart Rate Service: The Heart Rate Service is commonly used in fitness and health monitoring applications. It includes characteristics that provide real-time heart rate data, allowing central devices like smartphones or fitness trackers to monitor a user's heart rate during exercise.
+
+- Generic Access Profile (GAP): While not a service in itself, GAP defines the roles and procedures for device discovery and connection establishment in BLE. It plays a vital role in enabling devices to find and connect to each other seamlessly.
+
+***You can read more about Bluetooth® profiles [here](https://en.wikipedia.org/wiki/List_of_Bluetooth_profiles)***
+
+**Creating Custom Profiles** :
+
+In addition to standard profiles, developers have the flexibility to create custom profiles tailored to their specific application needs. These custom profiles define unique services and characteristics that match the requirements of a particular project.
+
+By using Bluetooth® LE profiles, developers can leverage existing standardized profiles for common applications or create custom profiles for specialized projects. This standardized approach simplifies the development process, enhances interoperability, and allows for the creation of diverse BLE-based applications, from health monitoring to home automation.
+
+As you explore Bluetooth® LE further, you'll discover a wide range of profiles designed to support various use cases. These profiles play a crucial role in ensuring that BLE devices can seamlessly communicate and provide valuable data to central devices.
+
+## ArduinoBLE Library
+
+The next part introduces how do establish a Bluetooth® LE connectionn, turn on the built in LED using your phone and read analog values via Bluetooth® LE. We will go through the following steps in order to create our sketch:
+
+- Create a new service.
+- Create an analog pin characteristic.
+- Set the name for our device.
+- Start advertising the device.
+- Create a conditional that works only if an external device is connected (smartphone).
+- Create a conditional that turns on an LED over Bluetooth®.
+- Read an analog pin over Bluetooth®.
+
+First, let's make sure we have the drivers for the board you are using installed. If we are using the Web Editor, we do not need to install anything. If we are using an offline editor, we need to install it manually. This can be done by navigating to **Tools > Board > Board Manager....**
```

Comment:
```suggestion
First, let's make sure we have the drivers for the board you are using installed. If we are using the Arduino Web Editor, we do not need to install anything. If we are using an offline editor, we need to install it manually. This can be done by navigating to **Tools > Board > Board Manager....**.
```